src/test/java/com/google/jenkins/plugins/credentials/oauth/GoogleRobotCredentialsTest.java

```diff
@@ -215,10 +212,26 @@ public void testGetById() throws Exception {
     assertNull(GoogleRobotCredentials.getById("not an id"));
   }
 
+  @LocalData
+  @Test
+  public void testMigration() {
+    /* LocalData contains an old credential with no id field and the project id my-google-project.
+    On deserialization the id should be filled with the project id. */
+    assertNotNull(GoogleRobotCredentials.getById(MIGRATION_PROJECT_ID));
```

Comment:
Do we want to adjust this test and add assert that credentials have Global scope after migration?
I also think it would be nice to have one test doing save credentials with System scope and then retrieve credentials by id and verify that Scope is System.

Reply:
After migration, not all credentials will have a Global scope because of the readResolve method